### ISSUE TITLE
AI research for newcomer to AI

### ISSUE THREAD

**Issue by Matthieu-Sauer** [awaketogether/AI#25]
**As a:** Member

**I want to:** Gain knowledge in the AI domain

**Description:**
The objective is to learn about AI creation / usage

**Definition of done:**
- [ ] Research the difference between Underfitting, good-fitting and overfitting
- [ ] Research what is Bias-variance trade-off
- [ ] Research what is Gradient descent in neural networks
- [ ] Research the difference between Sigmoid activation function and Rectified linear unit activation function
- [ ] Write documents containing the result of those reseach.

**Estimated workload:** 2 M/D
